docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -63,7 +63,9 @@ to satisfy the two above requirements.
 You can configure remote clusters globally by using
 <<cluster-update-settings,cluster settings>>, which you can update dynamically.
 Alternatively, you can configure them locally on individual nodes by using the
- `elasticsearch.yml` file.
+ `elasticsearch.yml` file. Note that the connection between the clusters uses
+the transport connection string (not the http connection) which is configured
+via the <<modules-transport,transport layer>>.
 
```

Comment:
This is redundant with the sentence at the beginning so I am not sure this is needed ?

Reply:
I think if I explicitly mention "host"/"port" as you suggested in your other comment, that would suffice. Thanks! :)